src/client/pydaos/raw/daos_cref.py

```diff
@@ -169,26 +169,16 @@ class DaosObjLayout(ctypes.Structure):
                 ("ol_nr", ctypes.c_uint32),
                 ("ol_shards", ctypes.POINTER(DaosObjShard * 5))]
 
-class CheckSum(ctypes.Structure):
-    _fields_ = [("cs_csum", ctypes.c_char_p),
-                ("cs_nr", ctypes.c_uint32),
-                ("cs_type", ctypes.c_uint16),
-                ("cs_len", ctypes.c_uint16),
-                ("cs_buf_len", ctypes.c_uint32),
-                ("cs_chunksize", ctypes.c_uint32)]
-
 class Extent(ctypes.Structure):
     _fields_ = [("rx_idx", ctypes.c_uint64),
                 ("rx_nr", ctypes.c_uint64)]
 
 class DaosIODescriptor(ctypes.Structure):
     _fields_ = [("iod_name", IOV),
-                ("iod_kcsum", CheckSum),
                 ("iod_type", ctypes.c_int),
                 ("iod_size", ctypes.c_uint64),
                 ("iod_nr", ctypes.c_uint32),
                 ("iod_recxs", ctypes.POINTER(Extent)),
-                ("iod_csums", ctypes.POINTER(CheckSum)),
                 ("iod_eprs", ctypes.c_void_p)]
```

Comment:
could we remove eprs while we are at it?

Reply:
I think we should remove iod_eprs? Otherwise, it may cause issues with the python code. Looks like this is removed from the C header.